src/fetcher/create-exercises.ts

```diff
@@ -67,6 +68,34 @@ export function createExercise(
   }
 }
 
+export function createSolution(uuid: Solution): FrontendSolutionNode {
```

Comment:
if FrontendSolutionNode is a property of FrontendExerciseNode, you can call this function from the method above.